### DISCUSSION TITLE
Sort new to old when looking at an actor appearances

### DISCUSSION THREAD

**Discussion by up2you1** [sct/overseerr#3092] [Feature Requests]
For Example, when I click into this actor, his appearances at the bottom are in a random order, would be good to have this order from newest to old.

![8B04EC87-1A15-4E28-8879-568FD0C29BD8](https://user-images.githubusercontent.com/35837260/197327738-d5bd9d9f-4724-41d3-bede-a8c816b8ed8b.jpeg)

thanks!

**Comment by sebwelch** [comment]
I had a look through the code. It seems that the order is based on the voteCount variable. The file is `src/components/PersonDetails/index.tsx` line 49.
I also checked the vote_count values for the various movies using the TMDB API. 

I think it would be a simple matter of changing the value the .tsx file uses to display the Movies & TV Shows.
I will have a play around with this.

**Reply by Zeron85** [reply]
Hi

Did you find away to get it to sort using release date?

**Comment by up2you1** [comment]
Any updates ?
